test_unstructured/partition/csv/test_csv.py

```diff
@@ -206,3 +206,13 @@ def test_partition_csv_respects_languages_arg():
     filename = "example-docs/stanley-cups.csv"
     elements = partition_csv(filename=filename, strategy="fast", languages=["deu"])
     assert elements[0].metadata.languages == ["deu"]
+
+
+def test_partition_csv_header():
+    filename = "example-docs/stanley-cups.csv"
+    elements = partition_csv(filename=filename, strategy="fast", include_header=True)
```

Comment:
lets parametrize this test so we test both positive `include_header=True` and negative `include_header=False` cases

Reply:
same with the csv test above

Reply:
Can I just change all the other test to use `include_header=False`. I want to keep the same structure which is already part of `test_xlsx.py` which does not parametrize this test. cf: https://github.com/Unstructured-IO/unstructured/blob/main/test_unstructured/partition/xlsx/test_xlsx.py#L54